### DISCUSSION TITLE
Will NuiTrack be updated for Unreal =< 5.4

### DISCUSSION THREAD

**Discussion by CaptainJeffo** [3DiVi/nuitrack-sdk#70] [General]
Today I could not find de NuiTrack plugin for the test version of unreal 5.4.   

Will 3DiVi update the  plugin? If yes, approx when?  

Thnks

**Comment by reuksv** [comment]
Hello @CaptainJeffo 
Yes, we will definitely update the plugin for Unreal 5.4. Within a week, as soon as the non-"preview" version is released.
(Maybe sooner if there are a lot of relevant requests)

**Reply by CaptainJeffo** [reply]
Thnks, thats great, I am looking foreward.
